be/tool/beck.c

```diff
@@ -419,18 +426,24 @@ static struct btype bt[] = {
 #undef _B
 
 enum {
-	MAX_GEN    	 =      256,
-	MAX_SCAN_QUEUED	 = 10000000,
-	MAX_QUEUED  	 =  1000000,
-	MAX_REC_SIZE     =  64*1024,
+	MAX_GEN    	         = 256,
+	MAX_SCAN_QUEUED	         = 10000000,
+	MAX_QUEUED  	         = 1000000,
+	MAX_REC_SIZE             = 64*1024,
+	/**
+	 * The value in MAX_GEN_DIFF_SEC is arrived on the basis of max time
+	 * difference between mkfs run on local and remote node and the
+	 * assumption that clock time delta between nodes is negligible.
+	 */
+	MAX_GEN_DIFF_SEC         = 30,
+	MAX_KEY_LEN              = 256,
+	MAX_VALUE_LEN            = 256,
+	DEFAULT_BE_SEG_LOAD_ADDR = 0x400000100000,
 	/**
-	 * This value is arrived on the basis of max time difference between
-	 * mkfs run on local and remote node and the assumption that time
-	 * difference between nodes is negligible.
+	 * The value of 44MB for DEFAULT_BE_MAX_TX_REG_SZwas picked from the
+	 * routine m0_be_ut_backend_cfg_default()
 	 */
-	MAX_GEN_DIFF_SEC =       30,
-	MAX_KEY_LEN      =      256,
-	MAX_VALUE_LEN    =      256
+	DEFAULT_BE_MAX_TX_REG_SZ = (44 * 1024 * 1024)
```

Comment:
Here, we might need to use:
DEFAULT_BE_MAX_TX_REG_SZ = (44 * 1024 * 1024ULL)

Currently, the value does not overflow the MAX_INT.
This value to assigned to a uint64_t. So, it's better to define it to ULL.

Reply:
Done.